<?xml version="1.0" encoding="UTF-8"?><testrun name="TestDao.testfindSysUserById" project="yycg-dao-0808" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testfindSysUserById" classname="cn.itcast.yycg.user.dao.TestDao" time="0.06">
    <error>org.springframework.orm.hibernate5.HibernateSystemException: Unable to locate persister: cn.itcast.yycg.domain.po.SysUser; nested exception is org.hibernate.UnknownEntityTypeException: Unable to locate persister: cn.itcast.yycg.domain.po.SysUser&#13;
	at org.springframework.orm.hibernate5.SessionFactoryUtils.convertHibernateAccessException(SessionFactoryUtils.java:219)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate.doExecute(HibernateTemplate.java:344)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate.executeWithNativeSession(HibernateTemplate.java:309)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate.get(HibernateTemplate.java:419)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate.get(HibernateTemplate.java:412)&#13;
	at cn.itcast.yycg.user.dao.impl.SysUserDaoImpl.findSysUserById(SysUserDaoImpl.java:22)&#13;
	at cn.itcast.yycg.user.dao.TestDao.testfindSysUserById(TestDao.java:20)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:606)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)&#13;
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:254)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:89)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)&#13;
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)&#13;
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:292)&#13;
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:193)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: org.hibernate.UnknownEntityTypeException: Unable to locate persister: cn.itcast.yycg.domain.po.SysUser&#13;
	at org.hibernate.internal.SessionFactoryImpl.locateEntityPersister(SessionFactoryImpl.java:797)&#13;
	at org.hibernate.internal.SessionImpl.locateEntityPersister(SessionImpl.java:2710)&#13;
	at org.hibernate.internal.SessionImpl.access$2500(SessionImpl.java:164)&#13;
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.&lt;init&gt;(SessionImpl.java:2648)&#13;
	at org.hibernate.internal.SessionImpl$IdentifierLoadAccessImpl.&lt;init&gt;(SessionImpl.java:2635)&#13;
	at org.hibernate.internal.SessionImpl.byId(SessionImpl.java:1102)&#13;
	at org.hibernate.internal.SessionImpl.get(SessionImpl.java:975)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate$1.doInHibernate(HibernateTemplate.java:426)&#13;
	at org.springframework.orm.hibernate5.HibernateTemplate.doExecute(HibernateTemplate.java:341)&#13;
	... 34 more&#13;
&#13;
</error>
  </testcase>
</testrun>
